social-network/src/components/ArticleImage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ArticleImage extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
```

Comment:
Nice job staying modular with all these components. While it's usually fine to go more modular than necessary, this is a scenario where it may have been easier to inline the img component.